src/public/assets/lang/nl-BE

```diff
@@ -0,0 +1,148 @@
+---
+# Global
+
+# footer.tsx
+Overview: Overzicht
+Home: Huis
+Features: Kenmerken
+downloads: downloads
+External links: Externe links
+Twitter:
+GitHub:
+YouTube:
+Discord:
+Reddit:
+Legal: Juridisch
+License: Licentie
+EULA:
+History of Incidents: Geschiedenis van incidenten
+Privacy and Security: Privacy en veiligheid
+Inspired by <%0|Avdan|%0>: Geïnspireerd doo <%0|Avdan|%0>
+AvdanOS contributors: AvdanOS medewerkers
+
+# alert.tsx
+<%0|AvdanOS|%0> is still in development.: AvdanOS is nog steeds in ontwikkeling
```

Comment:
A styling mark is missing here

Reply:
Hello Froxcey,

Thanks for pointing that flaw out... I will make sure to resolve it.